packages/api/Dockerfile

```diff
@@ -1,30 +1,13 @@
-FROM	node:16	as	builder
-WORKDIR	/app
+FROM node:18 as builder
 
-ENV	NODE_ENV development
-
-COPY	packages/api/package.json	yarn.lock	./
-RUN	yarn install --ignore-scripts --frozen-lockfile
-
-COPY	packages/api/tsconfig.json	.
-COPY	packages/api/src	src
-RUN	yarn run prepare
-
-FROM	node:16
 WORKDIR /app
+COPY . .
+RUN	yarn install --frozen-lockfile --ignore-scripts
+RUN yarn run prepare:lerna
```

Comment:
do we not need `NODE_ENV` here for builds?

Reply:
NODE_ENV=production was used to only install production dependencies for prod containers. The pkg version automatically only packages what's needed for production so it doesn't matter... unless we have any logic that depends on that, I suppose...

Reply:
Oh lol I guess we do ```
▶ git grep NODE_ENV | wc -l
23
```